### ISSUE TITLE
ci.tensorflow.org is now deprecated.  Where can I get nightly builds for pi-zero AND regular pis when I only have a pi3?

### ISSUE THREAD

**Issue by gragundier** [tensorflow/tensorflow#21469]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:Raspbian Stretch 9
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:Raspberry Pi 3 Model B
- **TensorFlow installed from (source or binary)**:latest pi-zero nightly binary that was on jenkins.
- **TensorFlow version (use command below)**:1.9.0 (ultimately not relevant so this should be good enough)
- **Python version**:3.5.3
- **Bazel version (if compiling from source)**:N/A
- **GCC/Compiler version (if compiling from source)**:N/A
- **CUDA/cuDNN version**:N/A
- **GPU model and memory**:VideoCore IV
- **Exact command to reproduce**:
pip3 install tf-nightly

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.
Now, that ci.tensorflow.org is gone.  I'm struggling to find an easy way to view changelogs between nightly builds AND download both pi-zero and regular raspberry pi nightly builds.  The reason why this is useful is because recently #21198 armv7 optimizations interferred with the tflite interpreter.  The current most practical work around was to go to ci.tensorflow.org and grab the pi-zero binary instead of the default one you get when installing tf-nightly or tensorflow on the raspberry pi.  I can see how this wouldn't be a problem for other tensorflow versions that only have one binary that works for only that specific machine.  I understand that you can use the 2.7 tensorflow vs 3.n tensorflow on other machines, but those you can do `pip install tensorflow` or `pip3 install tensorflow`.  This isn't possible with the raspberry pi as far as I can tell, or at least it's not documented clearly anywhere that I can.  Furthermore, it was great being able to roll back a couple of days and just download the .whl from yesterday with ease.  I saved many hours by just playing around with the available .whl's instead of having to cross-compile each one.  
tl;dr I miss ci.tensorflow.org.  Please let me choose whether to install the tf nightly binaries of either pi-zero or regular raspberry-pi by choice since they both work on the raspberry pi.
### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by aselle** [comment]
This should be easy now with the latest effort to make raspberry pi officially supported?
Does this address your needs? (not sure if it makes it easy to get the nightly?)
https://medium.com/tensorflow/tensorflow-1-9-officially-supports-the-raspberry-pi-b91669b0aa0

**Comment by gragundier** [comment]
No, there are two ways to compile for Raspberry Pi Non-zeroes (I'm talking 2,3,3b,3b+ etc): 

1. tensorflow/tools/ci_build/ci_build.sh PI tensorflow/tools/ci_build/pi/build_raspberry_pi.sh PI_ONE
2. CI_DOCKER_EXTRA_PARAMS="-e CI_BUILD_PYTHON=python3 -e CROSSTOOL_PYTHON_INCLUDE_PATH=/usr/include/python3.4" \
tensorflow/tools/ci_build/ci_build.sh PI-PYTHON3 tensorflow/tools/ci_build/pi/build_raspberry_pi.sh

These commands are taken directly from [the official tensorflow on raspbian webpage](https://www.tensorflow.org/install/install_raspbian).  The 2nd one has armv7 optimizations that currently break tflite's python interpreter_wrapper.  The 1st one which is what I believe ci.tensorflow.org used to compile for their nightly pi-zero build does not have those optimizations and thus can use tflite's interpreter.  Let's be honest, there really is no point to pip install tensorflow on the raspberry pi other than to use tflite.  Regular tensorflow is just way too bulky; you're better off just running it over wifi.  
tl;dr: ci.tensorflow.org offered two DIFFERENT binaries for raspberry pis: one without optimizations and one with.  Currently, optimizations do not work.  I could grab no optimization version as a prebuilt binary off of ci.tensorflow.org.  ci.tensorflow.org is down.  pip and github don't give me clear ways to pick and choose the binaries I need.  

ps.  I lost Jenkins once, I can't lose him a second time :crying_cat_face: my heart can't take it.

**Comment by freedomtan** [comment]
@gragundier if TF Lite related stuff compiled with optimization flags is what you need, I think something like
```
bazel build --config opt \
--local_resources 1024.0,0.5,0.5 \
--cxxopt=-mfpu=neon-vfpv4 \
--cxxopt=-ftree-vectorize \
--cxxopt=-funsafe-math-optimizations \
--cxxopt=-ftree-loop-vectorize \
--cxxopt=-fomit-frame-pointer \
//tensorflow/contrib/lite/tools/benchmark:benchmark_model
```
on your Pi devices works. Building the whole TensorFlow pip package on natively on Pi devices is painful. But building only TF Lite related code should be fine.

**Comment by freedomtan** [comment]
@gragundier I guess NO for TF Lite code. When building the pip package of TF natively on RPI 3, I found that some kernels, e.g., [concat](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/BUILD#L149-L170), use [`tf_kernel_library()`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tensorflow.bzl#L1075-L1179), which takes `--copt` but not `--cxxopt`.

**Comment by gunan** [comment]
The changes are about to be pushed to github.
We will have these under the README in the repository root.

**Comment by skeydan** [comment]
Hello, could anyone please provide the current URL for TF nightly builds, unrelated to raspberry PI?
Thank you!

**Comment by gunan** [comment]
Please see the README.md page at the repository root, as recommended above.

**Comment by skeydan** [comment]
thank you!
